pkgs/applications/audio/ams/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchgit
+, automake
+, alsaLib
+, ladspaH
+, libjack2
+, fftw
+, zita-alsa-pcmi
+, qt4
+, pkgconfig
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "ams";
+  version = "2.1.2";
+  rev = "a47d45272a1c387c1cfc37b9e82d6245429a3117";
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://git.code.sf.net/p/alsamodular/ams.git";
+    sha256 = "0v5lasrgghcjw47dfxzpxp93mxykimh3raz8kpzk2i7z7cnlp5wa";
+  };
+
+  nativeBuildInputs = [ pkgconfig autoreconfHook ];
+
+  buildInputs = [ alsaLib ladspaH libjack2 fftw zita-alsa-pcmi qt4 ];
+
+  # From Debian package
+  # TODO fetch properly from source, if possible
+  patches = [ ./Make-vocoder-module-compatible-to-C-11.patch ];
+
+  meta = with stdenv.lib; {
+    description = "Realtime modular synthesizer for ALSA";
+    homepage = http://alsamodular.sourceforge.net;
+    license = licenses.gpl2;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ stdenv.lib.maintainers.sjfloat ];
```

see my other comment, you'll need to be in the maintainers list, and then this will work fine :)
```suggestion
    maintainers = with maintainers; [ sjfloat ];
```

---

The clean up suggestion still applies.